buildSrc/src/main/groovy/com/github/okbuilds/core/dependency/DependencyCache.groovy

```diff
@@ -52,7 +52,31 @@ class DependencyCache {
     }
 
     private static void copyThirdPartyBuckFile(File dstDir) {
-        FileUtil.copyResourceToProject("thirdparty/BUCK_FILE", new File(dstDir, "BUCK"))
+        File buckFile = new File(dstDir, "BUCK")
+        buckFile <<
+"""
```

Comment:
moving here as classloader randomly fails to load the resource file `thirdparty/BUCK_FILE`